cmake/modules/RootCPack.cmake

```diff
@@ -29,7 +29,7 @@ string(REGEX REPLACE "^([0-9]+).*$" "\\1" CXX_MAJOR ${CMAKE_CXX_COMPILER_VERSION
 string(REGEX REPLACE "^([0-9]+)\\.([0-9]+).*$" "\\2" CXX_MINOR ${CMAKE_CXX_COMPILER_VERSION})
 
 #---Resource Files-----------------------------------------------------------------------------------
-configure_file(README/README README.txt COPYONLY)
+configure_file(README/DEVELOPMENT.md README.txt COPYONLY)
```

Comment:
@Axel-Naumann is that change what we want in general or we can drop this line in favor of the root.git/README.md?

Reply:
/README.md should be user facing, not focusing on developers. So IIUC this change makes sense? Thanks for finding this!

Reply:
@Axel-Naumann If we copy README.md into README.txt(like in the old code) in the build directory, we lose all the markdown formatting and there are some weird parts in the text, eg - image links. Should we leave it as a .md file instead? (I think if you double click a .md file, it shows the .txt version if you don't have a markdown viewer, so there shouldn't be any change in behaviour if we leave it as .md)

Reply:
Sorry right now I understand this makes no sense. We should not use this as the main README file, but only `/README.md` should be copied to the build directory. Just don't copy this one. I'd appreciate if you could verify that ROOT's main README.md gets copied over.

Reply:
Should be okay now :)